src/languages/swift.js

```diff
@@ -457,6 +457,37 @@ export default function(hljs) {
     end: /}/
   };
 
+  const TYPE_DECLARATION = {
+    match: [
```

Comment:
Please use `begin` since this is opening a scope (since you have `contains`), even if end is implied.